src/App.vue

```diff
@@ -422,6 +397,51 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     }
   }
 
+  private async populateNameRequest (filing: any): Promise<void> {
```

Comment:
Method name made me think it was doing something else. Suggest "processNameRequest", or ...?

Reply:
done